OAuthSwift/OAuthSwiftClient.swift

```diff
@@ -122,13 +122,14 @@ public class OAuthSwiftClient: NSObject {
             case .RequestURIQuery:
                 //Add oauth parameters as request parameters
                 request.parameters += self.credential.authorizationParametersWithSignatureForMethod(method, url: signatureUrl, parameters: signatureParameters, body: body)
+            case .CustomParameters:
+                break
```

Comment:
If I understand this is not Custom but no oauth parameters
A Custom will be something like `.CustomParameters([String:String])` or maybe dynamic one with closure `.CustomParameters(@autoclosure () -> [String:String])` (never tested)